docs/sphinx/user_guide/config_options.rst

```diff
@@ -89,8 +89,8 @@ and their default settings:
       ======================   ======================
       Variable                 Default
       ======================   ======================
-      ENABLE_TESTS             On 
-      ENABLE_EXAMPLES          On 
+      RAJA_ENABLE_TESTS        On 
+      RAJA_ENABLE_EXAMPLES     On 
```

Comment:
Should we add RAJA_ENABLE_BENCHMARKS to this?

Reply:
I don't think we need that. See David's comment above.

Reply:
ENABLE_BENCHMARKS is defined by BLT, so that is needed -- I'll make that change too.

Reply:
Done.